codejam3-pokemon/Program.cs

```diff
@@ -148,10 +147,42 @@ public static async Task GetPokemon(string pokemonspecies)
        
     }
 
-    public static void PrintPoke(string pokemonspecies){
+    public static async Task<bool>  isRealPoke(string pokemonspecies)
+    {
+        try
+        {
+            PokemonSpecies p = await DataFetcher.GetNamedApiObject<PokemonSpecies>(pokemonspecies);
+
+
+            return true;
+
+        }
+
+        catch (HttpRequestException e) when (e.StatusCode == System.Net.HttpStatusCode.NotFound)
+        {
+            Console.WriteLine("Pokemon species not found.");
+            return false;
+            
+        }
+
+    }
+    public static void quitGame() {
+        Console.WriteLine("Thanks for playing!");
+        System.Threading.Thread.Sleep(1500);System.Environment.Exit(0);
+            
+    }
+
+    public static bool PrintPoke(string pokemonspecies){
+        string[] knownPokes = { "jigglypuff", "charizard", "pikachu", "squirtle", "bulbasaur" };
+        string noArt = " _____  _______      _____      _____          __     _______          __    ___________                      .___\r\n  /  |  | \\   _  \\    /  |  |    /  _  \\________/  |_   \\      \\   _____/  |_  \\_   _____/___  __ __  ____    __| _/\r\n /   |  |_/  /_\\  \\  /   |  |_  /  /_\\  \\_  __ \\   __\\  /   |   \\ /  _ \\   __\\  |    __)/  _ \\|  |  \\/    \\  / __ | \r\n/    ^   /\\  \\_/   \\/    ^   / /    |    \\  | \\/|  |   /    |    (  <_> )  |    |     \\(  <_> )  |  /   |  \\/ /_/ | \r\n\\____   |  \\_____  /\\____   |  \\____|__  /__|   |__|   \\____|__  /\\____/|__|    \\___  / \\____/|____/|___|  /\\____ | \r\n     |__|        \\/      |__|          \\/                      \\/                   \\/                   \\/      \\/ ";
+        if (!knownPokes.Contains(pokemonspecies)) {
+            Console.WriteLine(noArt);
```

Comment:
this ascii art addition makes it clear to the user that we did not include ascii art for every pokemon.